app/main/controllers/shortcodes/RTMediaGalleryShortcode.php

```diff
@@ -74,7 +74,12 @@ static function register_scripts() {
 			$url_upload = substr( $url, 0, strrpos( $url, $rtmedia_slug ) );
 			$url        = trailingslashit( $url_upload ) . 'upload/';
 		} else {
-			$url = trailingslashit( $url ) . 'upload/';
+			if ( strstr( $url, '?' ) ) {
+				$url = explode( '?', $url );
+				$url = $url[0] . 'upload?' . trim( $url[1], '/' );
```

Comment:
missing forward slash after upload 
`https://yahil.rtdemo.in/upload?a=sds`
it should be
`https://yahil.rtdemo.in/upload/?a=sds`